res/values/strings.xml

```diff
@@ -892,6 +892,7 @@
     <string name="load_qr_code_as_image">Load QR Code as Image</string>
     <string name="qrscan_title">Scan QR Code</string>
     <string name="qrscan_hint">Hold your camera over the QR code.</string>
+    <string name="qrscan_hint_desktop">Move the QR code to the camera.</string>
```

Comment:
Maybe without the dot? 🙂

Reply:
yeah, it's better without

i copied the old one ... that could be changed as well as some point (be we should not mix changing strings and adding strings for $reasons)